src/main/webapp/app/overview/course-lectures/video-unit/video-unit.component.html

```diff
@@ -14,7 +14,7 @@ <h5 class="card-header unit-card-header" [class.rounded]="isCollapsed" (click)="
         </h5>
         <div class="card-body unit-card-body" [ngbCollapse]="isCollapsed">
             <div class="embed-responsive embed-responsive-16by9">
-                <iframe id="videoFrame" class="embed-responsive-item rounded" [src]="videoUrl" allow="fullscreen" allowfullscreen></iframe>
+                <iframe id="videoFrame" class="embed-responsive-item rounded" [src]="videoUrl" allow="fullscreen" allowfullscreen="1"></iframe>
```

Comment:
have you tried to add the following: `allowFullScreen="true" webkitallowfullscreen="true" mozallowfullscreen="true"` ?
You could also playaround with the `frameborder="0"` property.

Reply:
It was the feature policy after all. When testing it I just missted a `'` :facepalm:

Reply:
happens.. but well done 😀😉👍